src/devices/SocketCan/CanFlags.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.SocketCan
+{
+    [Flags]
+    public enum CanFlags : uint
```

Comment:
Can we add xmldoc comments to both the enum and the actual values?

Reply:
making this internal